main.go

```diff
@@ -594,7 +602,12 @@ func main() {
 		var response []ResponsePipelineExecution
 		for _, execution := range res {
 			logOutput := []string{}
-			tempLog := filepath.Join(os.TempDir(), execution.Executionid+".log")
+			logDir, exists := os.LookupEnv("ZETAFORGE_LOGS")
+			if !exists {
+				logDir = os.TempDir()
+			}
+			tempLog := filepath.Join(logDir, execution.Executionid+".log")
+
```

Comment:
Would it make more sense to set logDir at the start of main? Also I feel the check should be local vs cloud, but maybe I'm missing something.

Reply:
Not for this type of log, which is not the system log, but the log for the execution. I agree with you though, in the future we would want to do log handling but split our server logs from execution logs